lib/RuntimeTemplate.js

```diff
@@ -724,7 +724,7 @@ class RuntimeTemplate {
 				if (asiSafe) {
 					return `(0,${access})`;
 				} else {
-					return `Object(${access})`;
+					return asiSafe === false ? `;(0,${access})` : `Object(${access})`;
 				}
```

Comment:
mixed if and ?:

This could be made a bit more readable with only `if`s or only `?:`